warehouse/manage/views.py

```diff
@@ -969,6 +973,67 @@ def delete_macaroon(self):
         return HTTPSeeOther(redirect_to)
 
 
+@view_defaults(
+    route_name="manage.organizations",
+    renderer="manage/organizations.html",
+    uses_session=True,
+    require_csrf=True,
+    require_methods=False,
+    permission="manage:user",
+    has_translations=True,
+)
+class ManageOrganizationsViews:
+    def __init__(self, request):
+        self.request = request
+        self.user_service = request.find_service(IUserService, context=None)
+        self.organization_service = request.find_service(
+            IOrganizationService, context=None
+        )
+
+    @property
+    def default_response(self):
+        return {
+            "create_organization_form": CreateOrganizationForm(
+                organization_service=self.organization_service,
+            ),
+        }
+
+    @view_config(request_method="GET")
+    def manage_organizations(self):
+        return self.default_response
+
+    @view_config(request_method="POST", request_param=CreateOrganizationForm.__params__)
+    def create_organization(self):
+        form = CreateOrganizationForm(
+            self.request.POST,
+            organization_service=self.organization_service,
+        )
+
+        if form.validate():
+            data = form.data
+            organization = self.organization_service.add_organization(**data)
+            self.organization_service.add_catalog_entry(
+                organization.name, organization.id
+            )
+            self.organization_service.add_organization_role(
+                "Owner", self.request.user.id, organization.id
+            )
```

Comment:
This was my first stab at it:

https://github.com/pypa/warehouse/blob/3b373b452345aa4a289bb334bfa6f73378899516/warehouse/manage/views.py#L1015-L1019

I tried `self.request.user.username` to be consistent with additional data in other events:

https://github.com/pypa/warehouse/blob/3b373b452345aa4a289bb334bfa6f73378899516/warehouse/forklift/legacy.py#L1003-L1007

https://github.com/pypa/warehouse/blob/3b373b452345aa4a289bb334bfa6f73378899516/warehouse/forklift/legacy.py#L1008-L1016

https://github.com/pypa/warehouse/blob/3b373b452345aa4a289bb334bfa6f73378899516/warehouse/manage/views.py#L956-L963

https://github.com/pypa/warehouse/blob/3b373b452345aa4a289bb334bfa6f73378899516/warehouse/manage/views.py#L1159-L1163

I think username is immutable in PyPI so that should work okay?

Reply:
For the purposes of traceability we should use User IDs and reconstitute the user on display.

We _should_ also do this elsewhere, since username changes _are_ something that we want to support in the future, but that's not on you :)

Reply:
@divbzero to make this change to the `additional` field.

Reply:
Changed additional event data from `user.username` to `user.id` in [db0ec20](https://github.com/pypa/warehouse/pull/11184/commits/db0ec20ab3e83d78934b0a6ae3e58b65006eec92).